lib/web_ui/lib/src/engine/text/measurement.dart

```diff
@@ -147,8 +147,8 @@ class RulerManager {
   /// The returned ruler is marked as hit so there's no need to do that
   /// elsewhere.
   @visibleForTesting
-  ParagraphRuler findOrCreateRuler(ParagraphGeometricStyle style) {
-    ParagraphRuler ruler = _rulers[style];
+  ParagraphRuler? findOrCreateRuler(ParagraphGeometricStyle style) {
```

Comment:
In all cases, this function returns a ruler.

Reply:
Done.